src/components/traces/details/timeline/span.jsx

```diff
@@ -97,6 +96,8 @@ export default class Span extends React.Component {
             operationName
         } = span;
 
+        const serviceName = Span.getTagValue(span, 'service') || span.serviceName;
```

Comment:
interesting you are prioritizing tag over field.. is this common or a bug?

Reply:
> interesting you are prioritizing tag over field.. is this common or a bug?

This change is to visualize our indexer change (https://github.com/ExpediaDotCom/haystack-traces/pull/183/ in response to https://github.com/opentracing/specification/issues/131). Until there's a consensus on the OT specification, we're going to override the visualized service name with service tag, if it exists